public/app/plugins/panel/geomap/layers/basemaps/generic.ts

```diff
@@ -20,7 +20,9 @@ export const defaultXYZConfig: XYZConfig = {
 export const xyzTiles: MapLayerRegistryItem<XYZConfig> = {
   id: 'xyz',
   name: 'XYZ Tile layer',
+  description: 'Add map from custom source',
   isBaseMap: true,
+  showOpacity: true,
```

Comment:
Looking at this now... I think it would be better to have a `hideOpacity` flag, and only set if for 'markers' layer?  (everywhere else opacity seems useful)

Reply:
Changed.